pkg/ip/link.go

```diff
@@ -40,9 +41,15 @@ func makeVethPair(name, peer string, mtu int) (netlink.Link, error) {
 	return veth, nil
 }
 
-func makeVeth(name string, mtu int) (peerName string, veth netlink.Link, err error) {
+func makeVeth(namePrefix string, mtu int) (peerName string, veth netlink.Link, err error) {
 	for i := 0; i < 10; i++ {
-		peerName, err = RandomVethName()
+		peerName, err = RandomIfaceName(namePrefix)
```

Comment:
Could we get rid of RandomIfaceName and replace it with a refactored version of https://github.com/containernetworking/cni/blob/f6135c7ba38fe82545322407a629d671e90b3ecc/pkg/utils/utils.go#L31?